### PR TITLE
Added phpMD, phpCS & ESLint standards

### PR DESCRIPTION
**Pull request by Mvii7**

### CODE REVIEW

File: Test/Standards/Magento/PHPMD/FinalImplementation.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
```
Comment (cdiacon) [inline Test/Standards/Magento/PHPMD/FinalImplementation.php]: remove magento copyright

File: Test/Standards/Magento/PHPMD/design.xml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
```
Comment (cdiacon) [inline Test/Standards/Magento/PHPMD/design.xml]: remove magento copyright

File: Test/Standards/Magento/PHPMD/ruleset.xml [hunk 3]
```diff
@@ -0,0 +1,51 @@
+<?xml version='1.0' encoding="UTF-8"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
```
Comment (cdiacon) [inline Test/Standards/Magento/PHPMD/ruleset.xml]: remove magento copyright

File: Test/Standards/Magento/PHPMD/FinalImplementation.php (hunk 1 above)
Comment (Mvii7) [inline Test/Standards/Magento/PHPMD/FinalImplementation.php]: Done

File: Test/Standards/Magento/PHPMD/design.xml (hunk 2 above)
Comment (Mvii7) [inline Test/Standards/Magento/PHPMD/design.xml]: Done

File: Test/Standards/Magento/PHPMD/ruleset.xml (hunk 3 above)
Comment (Mvii7) [inline Test/Standards/Magento/PHPMD/ruleset.xml]: Done

File: composer.json [hunk 4]
```diff
@@ -3,14 +3,26 @@
...
+    }
   ],
   "require": {
     "php": "~5.5.0|~5.6.0|~7.0.0",
     "magento/framework": "100.*"
   },
+  "require-dev": {
+    "phpmd/phpmd": "@stable",
+    "magento/marketplace-eqp": "dev-master",
+    "squizlabs/php_codesniffer": "@stable"
```
Comment (cdiacon) [inline composer.json]: change to "squizlabs/php_codesniffer": "1.5.3" let's stick with versions used for core product

File: auth.json.sample [hunk 5]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (cdiacon) [inline auth.json.sample]: remove sample file

File: composer.json (hunk 4 above)
Comment (Mvii7) [inline composer.json]: Done

File: auth.json.sample (hunk 5 above)
Comment (Mvii7) [inline auth.json.sample]: Done

**Pull request merged**